scripts/aws_backup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+# Prerequisites:
+# 1. Configured aws client, new enough version needed
+# 2. S3 bucket
+#
+# Restore with command:
+# mongorestore --gzip --archive < <backup.gzip>
+
+set -e
+
+begin=$(date +%s)
+
+# DB host (secondary preferred as to avoid impacting primary performance)
+HOST=localhost
+
+# DB name
+DBNAME=db_name
+
+# S3 bucket name
+BUCKET=s3_bucket
+
+# Current time
+TIME=`/bin/date --iso-8601=seconds`
```

Comment:
maybe abs path is not needed for these tools - they probably are in path already.. could be even different in some distros..